### ISSUE TITLE
Larger electronics enclosure

### ISSUE THREAD

**Issue by lostapathy** [railcore/parts#11]
It would be quite helpful if the electronics enclosure on the 300ZL was larger.  There seems to be ample real estate to do it and I'm not sure of any compelling reason not to.

Doing so would allow us to make the wiring tidier as well as improve our options for better isolating/shielding the wiring (or at least terminals) that carry mains voltage.  I designed some parts for guarding the power supply terminals, but actual assembly was problematic in the current enclosure (issue #9 should help that, though).

It appears we could expand the enclosure 25-30mm toward the front and back and 50mm vertically lower and still fit comfortably, as well as still allow the cover panel to be cut from the waste piece out of the machine's front panel.

Obviously this could be made available in future panel sets by just cutting them for the new enclosure size.  We could also develop a retrofit kit that includes the updated electronics box panels as well as printable corners that use the new panels but still mount to the old holes so that older railcores don't need new mounting holes cut.

**Comment by kraegar** [comment]
The electronics panel is sized so it can easily and efficiently be cut out
of the scrap of the front panel.  I'm not positive of Joe's milling layout,
but enlarging it may actually end up increasing the kit costs due to more
HDPE use / waste, and additional milling time.

**Comment by lostapathy** [comment]
> The electronics panel is sized so it can easily and efficiently be cut out
of the scrap of the front panel.

Maybe I wasn't 100% clear as submitted, but I actually did measure mine and there's quite a bit of room for growing this cover and still cutting it from the waste.

It appears the electronics box was sized for the 250ZL and not scaled up when the 300ZL was designed.

**Comment by kraegar** [comment]
There's likely room to expand it some, but at least in how I milled it, it
wasn't as much as you'd think because you need good workholding on that
scrap in between the front panel layout and the Electronics panel. If you
can't hold that scrap securely the edges on the panels really shows it.
Especially with how narrow that front panel is already.  HDPE moving or
vibrating at all kills the surface finish. I can talk with Joe about it and
how his milling setup compares to what I did, but we'd need to be careful
to keep with something that doesn't cause problems.

You're right it was originally sized for the 250.  It was sized so it could
be laser cut out of the 250 scrap, that meant I couldn't mill it out of
that scrap, and had to use other material to do the 250's electronics
panels, which was a major headache.

**Comment by dandancheeseandham** [comment]
> The electronics panel is sized so it can easily and efficiently be cut out of the scrap of the front panel. I'm not positive of Joe's milling layout, but enlarging it may actually end up increasing the kit costs due to more HDPE use / waste, and additional milling time.

Oh! I did not realise that this was a design constraint! I guess because your photo, and a lot of photos from self-builders (so selection bias there admittedly) have all had acrylic electronics enclosures. Now I think about it, the kit builders do use all HDPE.
It makes sense to reduce waste.

I'm torn though, as a ZLT owner, it's also bloody awesome to have plenty of room 😄. If a bit more room can be eked out, (and just 1mm added to each side becomes 1324mm^2 in area) and I'm sure it would be appreciated by the new kit builders and self-builders alike.

**Comment by kraegar** [comment]
I think we can probably get a bit more room.  We'll also have to work with
Mandala and Printed Solid, though, as this would complicate their sales of
the after-market panels.

**Comment by dandancheeseandham** [comment]
Yeah. Understood. I think it is realised that this is more of an involved upgrade than most.

**Comment by lostapathy** [comment]
> Yeah. Understood. I think it is realised that this is more of an involved upgrade than most.

Correct - that's why I tagged as "railcore 2.5" as per the discussion on #4
